### PR TITLE
Webapp manage projects pages

### PR DESCRIPTION
**Pull request by IstvanM**
Reorganized slightly the pages structure.
Added:
- Connections Pages
- Project Pages
- Users Pages

### CODE REVIEW

File: mitzu/adapters/sqlalchemy_adapter.py [hunk 1]
```diff
@@ -225,6 +225,16 @@ def _generate_retention_series_cte(
...
 
+    def list_schemas(self) -> List[str]:
+        engine = self.get_engine()
+        insp = SA.inspect(engine)
+        return insp.get_schema_names()
+
+    def list_tables(self, schema: str) -> List[str]:
+        engine = self.get_engine()
+        insp = SA.inspect(engine)
+        return insp.get_table_names(schema=schema)
```
Comment (medgyes) [inline mitzu/adapters/sqlalchemy_adapter.py:236]: probably it won't cause any issues at the web discovery, but these methods may not make any sense for a file adapter

Comment (IstvanM) [reply mitzu/adapters/sqlalchemy_adapter.py:236]: File adatpter is in fact a Sqlite adapter, in theory it should work there as well. I will check though.

File: mitzu/visualization/tooltips.py [hunk 2]
```diff
@@ -74,40 +74,40 @@ def get_segmentation_tooltip(
...
+        + pdf[C.COLOR_COL].astype(str)
+        + f"<br /><br /><b>{agg_type_str}: </b>"
+        + round(pdf[C.Y_AXIS_COL], 2).astype(str)
+        + f" {value_suffix}"
+        + "<br /><b>Initial user count: </b>"
+        + pdf[GA.USER_COUNT_COL + "_1"].astype(str)
+        + "<br /><b>Retaining user count: </b>"
+        + pdf[GA.USER_COUNT_COL + "_2"].astype(str)
+    )
+    # else:
```
Comment (IstvanM) [inline mitzu/visualization/tooltips.py:95]: Delete

File: mitzu/webapp/navbar.py [hunk 3]
```diff
@@ -40,17 +39,19 @@ def create_mitzu_navbar(
...
-    @callback(
-        Output(NAVBAR_COLLAPSE, "is_open"),
-        [Input({"type": NAVBAR_TOGGLER, "index": ALL}, "n_clicks")],
-        [State(NAVBAR_COLLAPSE, "is_open")],
-    )
-    def toggle_navbar_collapse(n, is_open):
-        if n:
-            return not is_open
-        return is_open
+    # @callback(
```
Comment (IstvanM) [inline mitzu/webapp/navbar.py:47]: Delete

File: mitzu/webapp/navbar.py [hunk 4]
```diff
@@ -3,12 +3,11 @@
...
 
 import dash_bootstrap_components as dbc
-from dash import html, callback
-from dash.dependencies import State, Input, Output, ALL
+from dash import html
 from typing import List
 
-NAVBAR_COLLAPSE = "navbar-collapse"
-NAVBAR_TOGGLER = "navbar-toggler"
+# NAVBAR_COLLAPSE = "navbar-collapse"
```
Comment (IstvanM) [inline mitzu/webapp/navbar.py:9]: Delete

**Pull request closed without merging**